core/src/com/unciv/models/ruleset/Speed.kt

```diff
@@ -71,7 +71,7 @@ class Speed : RulesetObject() {
         yield(FormattedLine("Golden age length modifier: [${goldenAgeLengthModifier * 100}]%${Fonts.happiness}"))
         yield(FormattedLine("Adjacent city religious pressure: [$religiousPressureAdjacentCity]${Fonts.faith}"))
         yield(FormattedLine("Peace deal duration: [$peaceDealDuration] turns${Fonts.turn}"))
-        yield(FormattedLine("Start year: " + ("[${abs(startYear).toInt()}] " + (if (startYear < 0) "BC" else "AD")).tr()))
+        yield(FormattedLine("Start year: [" + ("{[${abs(startYear).toInt()}] " + (if (startYear < 0) "BC" else "AD") + "}]").tr()))
```

Comment:
Until the nesting PR is accepted this won't work, so I'm putting this off for now (we could just resolve the terms with .tr() instead of curly braces though so I'm not sure if this trick is needed)

Reply:
I did see that, but if there's something that I approved but did not merge I like to be specific about why not :)